### ISSUE TITLE
RKE2 with containerd and differing containerd.sock location

### ISSUE THREAD

**Issue by padelt** [ssup2/kpexec#10]
In my K8s cluster, the socket file for accessing the container runtime is at `/run/k3s/containerd/containerd.sock`.
Calling kpexec fails like this:
```bash
$ kubectl kpexec -it mypod-0 -- id
Defaulting container name to mypod.
Create cnsenter pod (cnsenter-fmh41m2qi4)
Wait to run cnsenter pod (cnsenter-fmh41m2qi4)
Failed to wait running cnsenter pod (cnsenter-fmh41m2qi4)
Print cnsenter pod (cnsenter-fmh41m2qi4) events
---
MountVolume.SetUp failed for volume "containerd-socket" : hostPath type check failed: /run/containerd/containerd.sock is not a socket file
---
Delete cnsenter pod (cnsenter-fmh41m2qi4)
```

I can see the location in https://github.com/ssup2/kpexec/blob/master/pkg/cmd/kpexec/kpexec.go#L54 (with `contRootContdPath` being another relevant constant two lines below).
Are these values hardcoded or can I change them from the command line or in a configuration file?

**Comment by ssup2** [comment]
@padelt Now it's hardcoded. I will add a option for this. Thanks :)

**Comment by padelt** [comment]
That would be great! :-)
I'd be happy to test - just let me know.

**Comment by ssup2** [comment]
@padelt sorry too late.
I released v0.4.0 and it support set socket path.
please try again with `--cri /run/k3s/containerd/containerd.sock` Option?
